packages/@atjson/renderer-commonmark/src/index.ts

```diff
@@ -221,28 +222,26 @@ function codify(line: string) {
 }
 
 export default class CommonmarkRenderer extends Renderer {
-  /**
-   * Controls whether HTML entities should be escaped. This
-   * may be desireable if markdown is being generated for humans
-   * and your markdown flavor doesn't support HTML.
-   *
-   * By default, `escapeHtmlEntities` is set to `true` if your
-   * schema has an annotation with the type `html`. You can override
-   * this configuration by passing in another parameter to the constructor.
-   */
-  protected options: {
-    escapeHtmlEntities: boolean;
-  };
-
   protected state: any;
 
   constructor(
     document: Document | { text: string; marks: Mark[]; blocks: Block[] },
-    options: { escapeHtmlEntities: boolean } = { escapeHtmlEntities: true }
+    /**
+     * Controls whether HTML entities should be escaped. This
+     * may be desireable if markdown is being generated for humans
+     * and your markdown flavor doesn't support HTML.
+     *
+     * By default, `escapeHtmlEntities` is set to `true` if your
+     * schema has an annotation with the type `html`. You can override
+     * this configuration by passing in another parameter to the constructor.
+     */
+    protected options: {
```

Comment:
Wondering why this was moved from above to inline? I understand that there's less duplication but it does kinda hide that this is hoisted